### ISSUE TITLE
A Small Typo on the Website

### ISSUE THREAD

**Issue by lambduli** [effekt-lang/effekt-website#32]
There is a small typo on the website. In the `Home` section, `Effect Safety` sub-section there is this sentence:

```
Effect handlers are great, but it easy to forget handling an effect.
```

There is a missing `is`.

It's this line: https://github.com/effekt-lang/effekt-website/blob/master/_layouts/home.html#L140

I could make a pull request but I figured I'll ask first - maybe you'd not appreciate a PR for such a small thing.

Take care.

**Comment by b-studios** [comment]
Hey, thanks for noticing. We always appreciate PRs :)

**Comment by b-studios** [comment]
Closed by #33
